catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -512,18 +511,7 @@ public void setMetacardTypes(List<MetacardType> metacardTypes) {
     }
 
     private Predicate<Metacard> isVersionOrDeleted() {
-        return (m) -> {
-            String metacardTypeName = m.getMetacardType()
-                    .getName();
-
-            String metacardVersionTypeName = MetacardVersionImpl.getMetacardVersionType()
-                    .getName();
-            String deletedMetacardTypeName = DeletedMetacardImpl.getDeletedMetacardType()
-                    .getName();
-
-            return metacardVersionTypeName.equals(metacardTypeName)
-                    || deletedMetacardTypeName.equals(metacardTypeName);
-        };
+        return (m) -> MetacardVersionImpl.isVersion(m) || DeletedMetacardImpl.isDeleted(m);
```

Comment:
✏️ We could define this as a `Predicate` constant instead of a method returning a `Predicate`.